eng/Version.Details.xml

```diff
@@ -65,6 +65,10 @@
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
       <Sha>5535e31a712343a63f5d7d796cd874e563e5ac14</Sha>
     </Dependency>
+    <Dependency Name="System.Formats.Asn1" Version="8.0.1">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
+      <Sha>bf5e279d9239bfef5bb1b8d6212f1b971c434606</Sha>
```

Comment:
> * How should the SHA be obtained? I suppose for runtime I just grab SHA for appropriate tag - e.g. https://github.com/dotnet/runtime/tree/v8.0.1, right?

This is incorrect for subtle reasons. _Packages_ that build in `dotnet/runtime` rev their versions and republish _only_ when they change meaningfully, but _the overall runtime_ revs version every time. This means that 8.0.1 of some package may have been built in _any_ 8.0.x _of runtime_. In this case, https://nuget.info/packages/System.Formats.Asn1/8.0.1 says that https://github.com/dotnet/runtime/commit/2aade6beb02ea367fd97c4070a4198802fe61c03 is the relevant commit, which is https://github.com/dotnet/runtime/releases/tag/v8.0.7, so "the first time ASN1 8 had to be serviced was the seventh overall servicing release of .NET 8".

```suggestion
      <Sha>2aade6beb02ea367fd97c4070a4198802fe61c03</Sha>
```

Reply:
This is super helpful info! Thanks!